src/http/http_server.h

```diff
@@ -100,7 +100,31 @@ class http_service
 
     virtual std::string path() const = 0;
 
-    void register_handler(std::string path, http_callback cb, std::string help);
+    void register_handler(std::string sub_path, http_callback cb, std::string help);
```

Comment:
```suggestion
    void register_handler(const char *sub_path, http_callback cb, const char *help);
```

Reply:
There may be very little performance improvement, and we have some methods to avoid memory copy. C++ style would be more readable IMO.